scripts/build-windows-k8s.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+set -eo pipefail
+
+ACS_ENGINE_HOME=${GOPATH}/src/github.com/Azure/acs-engine
+
+version=""
+acs_patch_version=""
+winnat="false"
+
+while getopts "hv:p" opt; do
+  case $opt in
+    h)
+      echo "$0 [-v version] [-p acs_patch_version] [-w]"
+      echo " -v <version>: version"
+      echo " -p <patched version>: acs_patch_version"
+      exit 0
+      ;;
+    v)
+      version="${OPTARG}"
+      ;;
+    p)
+      acs_patch_version="${OPTARG}"
+      ;;
+    \?)
+      echo "$0 [-v version] [-p acs_patch_version] [-w]"
+      exit 1
+      ;;
+  esac
+done
+
+if [[ -z $version ]]; then
+	echo "Unknown or no Kubernetes version provided"
+	exit
+fi
+
+if [[ -z $acs_patch_version ]]; then
```

Comment:
I'm not sure I follow...if you do this:

`build_windows_k8s.sh -v 1.7.2 -p 1`

this will create artifacts in `_dist/k8s-windows-v1.7.2-1`. Is there something else we should do?